apps/test/util/testUtils.js

```diff
@@ -161,6 +161,10 @@ exports.debugMode = function () {
  * @param {number} top Vertical offset from top left of visualization to drop at
  */
 exports.dragToVisualization = function (type, left, top) {
+  // this delta of 40 is a bit of a mystery, but I think it has to do with our
+  // tests not having CSS applied to them.
+  left = left + 40;
```

Comment:
CSS definitely is not applied - or at the very least it's a different set of CSS than we see in our app when run in dashboard (which is a little bit disconcerting). I arguably should dig into this 40 pixels a little more to understand exactly what's happening. I was getting a little frustrated with phantomjs while trying to get the simulated drag working, and I think I just wanted to be done. I'll poke around a little more/better tomorrow.

I'm not too sure what's happening with the scale. When I looked at the DOM, it did not have `transform: scale(0.5)` set. Might be worth me digging into this a little bit more too. Thanks for the review.